obspy/imaging/waveform.py

```diff
@@ -1300,6 +1308,10 @@ def __sect_init_plot(self):
             else:
                 raise NotImplementedError("sect_orientiation '%s' is not "
                                           "valid." % self.sect_orientation)
+            if self.wiglets:
+                fillfun(time, data, self._tr_offsets_norm[_tr], where=data > self._tr_offsets_norm[_tr], facecolor='blue')
+                fillfun(time, data, self._tr_offsets_norm[_tr], where=data < self._tr_offsets_norm[_tr], facecolor='red')
```

Comment:
I like this. If I am not mistaken then to load the parameters from the kwargs an `update` on the dict would suffice to do the trick.